design/closeups/wifi_router.md

```diff
@@ -2,10 +2,10 @@
 
 #closeup 
 
-A [upper_hall_wifi_router](../items/upper_hall_wifi_router.md) sits in a free space under a wooden floor. A loose board lies next to it. There are cables going into the sides and there are two antennas at the top.
+A [upper_hall_Wi-Fi_router](../items/upper_hall_wifi_router.md) sits in a free space under a wooden floor. A loose board lies next to it. There are cables going into the sides, and two antennas at the top.
 
-On the front is a vendor logo. It's two rectangles side by side (symbolizing the [Leverkusen Rhine Bridge](https://de.wikipedia.org/wiki/Rheinbr%C3%BCcke_Leverkusen)) and the name "Levku". Under the logo sits the model name "AirComm 1155". In the lower right corner, we see LEDs showing Power, Connect, LAN and WIFI. Power and Connect are static green, LAN is off and WIFI is blinking irregularly.
+On the front is a vendor logo. It has two rectangles side by side (symbolizing the [Leverkusen Rhine Bridge](https://de.wikipedia.org/wiki/Rheinbr%C3%BCcke_Leverkusen)) and the name "Levku". Under the logo sits the model name "AirComm 1155". In the lower right corner, we see LEDs showing Power, Connect, LAN and Wi-Fi. Power and Connect are static green, LAN is off and Wi-Fi is blinking irregularly.
```

Comment:
I wanted to say, that the vendor logo *consists* of two rectangles. Is that the same as to use *has*?

Reply:
Then I'd say that "has" is suitable, just as the (detestable) olympic logo has seven rings. However, it might be better to say that the logo _comprises_ two rectums.

Reply:
Okay then.